src/common/filament.h

```diff
@@ -25,10 +25,10 @@ typedef enum {
     FILAMENTS_END
 } FILAMENT_t;
 
-#define DEFAULT_FILAMENT FILAMENT_PLA
+static const FILAMENT_t DEFAULT_FILAMENT = FILAMENT_PLA;
 
-#define PREHEAT_TEMP        170.01f // like 170 but user cannot set it
-#define PREHEAT_TEMP_STRING "170"   // used for G codes
+static const uint8_t PREHEAT_TEMP = 170; // like 170 but user cannot set it
```

Comment:
the previous define resulted in a float value, this new expression is onlu uint8_t. Is that ok?

Reply:
The precise value does not matter since the detection is not done by specific value.